pkgs/os-specific/linux/dpdk-kmods/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchurl, kernel }:
+
+stdenv.mkDerivation rec {
+  pname = "dpdk-kmods";
+  version = "2021-04-21";
+
+  src = fetchurl {
+    url = "http://git.dpdk.org/dpdk-kmods/snapshot/dpdk-kmods-e13d7af77a1bf98757f85c3c4083f6ee6d0d2372.tar.xz";
```

Comment:
Can we use fetchGit?

Reply:
I should probably use fetchzip here actually. fetchzip should be actually faster than fetchgit.